app/scripts/components/exploration/hooks/use-stac-metadata-datasets.ts

```diff
@@ -109,6 +114,22 @@ async function fetchStacDatasetById(
       ...commonTimeseriesParams,
       domain: featuresApiData.extent.temporal.interval[0]
     };
+  } else if (type === 'wms') {
+    let domain = data.summaries?.datetime?.[0]
+      ? data.summaries.datetime
+      : data.extent.temporal.interval[0];
+
+    // @TODO: what to do with timeless data? Setting up as today as a temporary solution
```

Comment:
Yeah, exactly. All other UIs are designed with the assumption that the dataset has timestamps. I am concerned that if it is shipped as it is now, this will give users a wrong impression. What is the plan to put ArcGIS data in production?

Reply:
we need to do it soon in some of the other instances, although those don't use the "timeless" data for now. but we need to think about it